src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public Figure getRandomFigure() {
+        Random random = new Random();
+        ColorSupplier colorSupplier = new ColorSupplier();
+        int figure = random.nextInt(5);
```

Comment:
don't hardcode 5, get this number from enum values

Reply:
not fixed, make it a constant value if you don't have enum with figure types